clusterloader2/pkg/measurement/common/etcd_metrics.go

```diff
@@ -75,6 +75,7 @@ func (e *etcdMetricsMeasurement) Execute(config *measurement.MeasurementConfig)
 	}
 
 	etcdInsecurePort := config.ClusterFramework.GetClusterConfig().EtcdInsecurePort
+	isSSHSupported := config.ClusterFramework.GetClusterConfig().IsSSHSupported
```

Comment:
Hmm, I think that if the ssh is not supported then this measurement has nothing to do. In such cases we should just log and info and return from the Execute method. Does it make sense?